android/AndroidManifest.xml

```diff
@@ -664,8 +664,7 @@
 
     <activity
         android:name="app.organicmaps.help.HelpActivity"
-        android:configChanges="orientation|screenLayout|screenSize"
```

Comment:
It means that the activity will be recreated on screen rotation, and users will see a blank screen for a moment, more visible on older devices. Why is it mandatory for this PR?

Reply:
At least some TODO can be left here. Intentionally making the app slower does not fit well into Organic Maps "we are the fastest" approach.

Reply:
TODO to enable it back when layout will not be recreated.